src/Thorn_Calculus-Processes.thy

```diff
@@ -353,6 +353,27 @@ lemma environment_dependent_parallel:
   \<^theory_text>\<open>Thorn_Calculus-Communication\<close>.
 *)
 
+subsection \<open>Repeated Receiving\<close>
+
+corec repeated_receive :: "chan family \<Rightarrow> (val \<Rightarrow> process family) \<Rightarrow> process family" where
+  "repeated_receive A \<P> e = Receive (A e) (\<lambda>x. Parallel (\<P> x e) (repeated_receive A \<P> e))"
+syntax
+  "_repeated_receive" :: "chan family \<Rightarrow> pttrn \<Rightarrow> process family \<Rightarrow> process family"
+  (\<open>(3_ \<triangleright>\<^sup>\<infinity> _./ _)\<close> [53, 0, 52] 52)
+translations
+  "A \<triangleright>\<^sup>\<infinity> x. P" \<rightleftharpoons> "CONST repeated_receive A (\<lambda>x. P)"
+print_translation \<open>
+  [preserve_binder_abs_receive_tr' @{const_syntax repeated_receive} @{syntax_const "_repeated_receive"}]
```

Comment:
This line exceeds the maximum line length.

Reply:
True. However, the whole code isn’t polished yet. I’ll deal with the problem of this overlong line as part of the [project that also this pull request is part of](https://github.com/input-output-hk/thorn-calculus/projects/1).

Reply:
Great.